sql_user_check_dynamodb/run.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+
+if ! command -v "aws" &> /dev/null
+then
+    echo "Command aws could not be found, Follow https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html to install aws"
+    exit 1
+fi
+
+get_table_info() {
+    local region=$1
+    local mode=$2
+
+    total_records=0
+    table_count=0
+    tables=$(aws dynamodb list-tables --region "$region" --query "TableNames" --output text)
+
+    for table in $tables; do
+        table_count=$((table_count + 1))
+        count=$(aws dynamodb scan --table-name "$table" --region "$region" --select "COUNT" --query "Count")
```

Comment:
This uses RCUs. Customers will not want to spend RCU just for a permission script. Can we use https://docs.aws.amazon.com/cli/latest/reference/dynamodb/describe-table.html and use item count? A wrong or approx count is fine here

Reply:
done.